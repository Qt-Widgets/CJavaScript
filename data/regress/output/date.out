Sat Aug 27 08:10:57 2016
Sat Dec  6 22:28:37 1969
getDate 6
getDay 6
getFullYear 1969
getHours 22
getMilliseconds 0
getMinutes 28
getMonth 11
getSeconds 37
getTime -2147483648
getTimezoneOffset 0
getUTCDate 6
getUTCDay 6
getUTCFullYear 1969
getUTCHours 22
getUTCMilliseconds 0
getUTCMinutes 28
getUTCMonth 11
getUTCSeconds 37
getYear 69
Sat Dec  6 22:28:37 1969
Sun Dec  6 22:28:37 65
Sun Dec  6 11:28:37 65
Sun Dec  6 11:28:37 65
Sun Dec  6 11:45:37 65
Fri Mar  6 11:45:37 65
Fri Mar  6 11:45:32 65
Fri Mar  6 11:45:32 65
Fri Mar  6 11:45:32 65
Fri Mar  6 11:45:32 65
Fri Mar  6 11:45:32 65
Fri Mar  6 11:45:32 65
Fri Mar  6 11:45:32 65
Fri Mar  6 11:45:32 65
Fri Mar  6 11:45:32 65
Mon Mar  6 11:45:32 1865
toDateString Mon Mar 06 1865
toGMTString Mon Mar  6 11:45:32 1865
toISOString 1865-03-06TMar:45:-3307850306.0Z
toJSON 1865-03-06TMar:45:-3307850306.0Z
toLocalDateString Mon Mar 06 1865
toLocalFormat Mon Mar  6 11:45:32 1865
toLocalTimeString Mar:45:-3307850306 LMT
toString Mon Mar  6 11:45:32 1865
toTimeString Mar:45:-3307850306 LMT
toUTCString Mon Mar  6 11:45:32 1865
valueOf -3307850306000
