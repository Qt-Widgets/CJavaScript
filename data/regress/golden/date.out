Fri Aug 26 2016 17:39:56 GMT-0400 (EDT)
Sat Dec 06 1975 00:00:00 GMT-0500 (EST)
getDate 6
getDay 6
getFullYear 1975
getHours 0
getMilliseconds 0
getMinutes 0
getMonth 11
getSeconds 0
getTime 187074000000
getTimezoneOffset 300
getUTCDate 6
getUTCDay 6
getUTCFullYear 1975
getUTCHours 5
getUTCMilliseconds 0
getUTCMinutes 0
getUTCMonth 11
getUTCSeconds 0
getYear 75
Wed Dec 24 1975 00:00:00 GMT-0500 (EST)
Fri Dec 24 1965 00:00:00 GMT-0500 (EST)
Fri Dec 24 1965 11:00:00 GMT-0500 (EST)
Fri Dec 24 1965 11:00:00 GMT-0500 (EST)
Fri Dec 24 1965 11:45:00 GMT-0500 (EST)
Wed Mar 24 1965 11:45:00 GMT-0400 (EDT)
Wed Mar 24 1965 11:45:32 GMT-0400 (EDT)
Wed Dec 31 1969 19:00:00 GMT-0500 (EST)
Fri Jan 23 1970 19:00:00 GMT-0500 (EST)
Sat Jan 23 1965 19:00:00 GMT-0500 (EST)
Sun Jan 24 1965 06:00:00 GMT-0500 (EST)
Sun Jan 24 1965 06:00:00 GMT-0500 (EST)
Sun Jan 24 1965 06:45:00 GMT-0500 (EST)
Wed Mar 24 1965 07:45:00 GMT-0400 (EDT)
Wed Mar 24 1965 07:45:32 GMT-0400 (EDT)
Fri Mar 24 1865 07:45:32 GMT-0400 (EDT)
toDateString Fri Mar 24 1865
toGMTString Fri, 24 Mar 1865 11:45:32 GMT
toISOString 1865-03-24T11:45:32.005Z
toJSON 1865-03-24T11:45:32.005Z
toLocalDateString 3/24/1865
toLocalFormat 3/24/1865, 6:49:30 AM
toLocalTimeString 6:49:30 AM
toString Fri Mar 24 1865 07:45:32 GMT-0400 (EDT)
toTimeString 07:45:32 GMT-0400 (EDT)
toUTCString Fri, 24 Mar 1865 11:45:32 GMT
valueOf -3306312867995
